### PR TITLE
Scheduler: write description for the Virtual Scrolling demo

### PR DESCRIPTION
**Pull request by pavel-2020**

### CODE REVIEW

**Review by RomanTsukanov — DISMISSED** [review]

**Review by AntonSermyazhko — DISMISSED** [review]

File: JSDemos/Demos/Scheduler/VirtualScrolling/description.md [hunk 1]
```diff
@@ -1 +1,5 @@
-TODO
\ No newline at end of file
+Virtual scrolling helps increase the **Scheduler**'s performance on large datasets. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```
Comment (Abadzhev) [inline JSDemos/Demos/Scheduler/VirtualScrolling/description.md]:
```suggestion
If a **Scheduler** is bound to a large data set, enable Virtual Scrolling to optimize the widget's performance, In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```

File: JSDemos/Demos/Scheduler/VirtualScrolling/description.md [hunk 2]
```diff
@@ -1 +1,5 @@
-TODO
\ No newline at end of file
+If a **Scheduler** is bound to a large data set, enable Virtual Scrolling to optimize the widget's performance. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Scheduler/VirtualScrolling/description.md]: We don't usually capitalize feature names.

```suggestion
If a **Scheduler** is bound to a large data set, enable virtual scrolling to optimize the widget's performance. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: JSDemos/Demos/Scheduler/VirtualScrolling/description.md (hunk 2 above)
Comment (arminal) [reply JSDemos/Demos/Scheduler/VirtualScrolling/description.md]:
```suggestion
If a **Scheduler** is bound to a large data set, enable virtual vcrolling to optimize widget performance. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```

File: JSDemos/Demos/Scheduler/VirtualScrolling/description.md [hunk 3]
```diff
@@ -1 +1,5 @@
...
+
+To enable the virtual scrolling mode, set the [scrolling](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling).[mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option to *"virtual"* as done in this demo.
```
Comment (arminal) [inline JSDemos/Demos/Scheduler/VirtualScrolling/description.md]:
```suggestion
To enable virtual scrolling mode, set the [scrolling](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling).[mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option to *"virtual"* as it is shown in this demo.
```

File: JSDemos/Demos/Scheduler/VirtualScrolling/description.md [hunk 4]
```diff
@@ -1 +1,5 @@
...
+
+Virtual scrolling is supported by the following [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/): *"day"*, *"week"*, and *"workWeek"*.
```
Comment (arminal) [inline JSDemos/Demos/Scheduler/VirtualScrolling/description.md:5]: workWeek - is that a typo?

Comment (pavel-2020) [reply JSDemos/Demos/Scheduler/VirtualScrolling/description.md:5]: No, it is a view: https://js.devexpress.com/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Week_and_WorkWeek_Views

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**